src/Compilers/VisualBasic/Portable/PublicAPI.Unshipped.txt

```diff
@@ -52,7 +52,7 @@ Overrides Microsoft.CodeAnalysis.VisualBasic.VisualBasicSyntaxRewriter.VisitTupl
 Overrides Microsoft.CodeAnalysis.VisualBasic.VisualBasicSyntaxRewriter.VisitTupleType(node As Microsoft.CodeAnalysis.VisualBasic.Syntax.TupleTypeSyntax) -> Microsoft.CodeAnalysis.SyntaxNode
 Shared Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.TupleElement(identifierName As Microsoft.CodeAnalysis.VisualBasic.Syntax.IdentifierNameSyntax, asKeyword As Microsoft.CodeAnalysis.SyntaxToken, type As Microsoft.CodeAnalysis.VisualBasic.Syntax.TypeSyntax) -> Microsoft.CodeAnalysis.VisualBasic.Syntax.TupleElementSyntax
 Shared Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.TupleElement(identifierName As Microsoft.CodeAnalysis.VisualBasic.Syntax.IdentifierNameSyntax, type As Microsoft.CodeAnalysis.VisualBasic.Syntax.TypeSyntax) -> Microsoft.CodeAnalysis.VisualBasic.Syntax.TupleElementSyntax
-Shared Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.TupleElement(type As Microsoft.CodeAnalysis.VisualBasic.Syntax.TypeSyntax) -> Microsoft.CodeAnalysis.VisualBasic.Syntax.TupleElementSyntax
+Shared Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.TupleElement() -> Microsoft.CodeAnalysis.VisualBasic.Syntax.TupleElementSyntax
```

Comment:
>Shared Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.TupleElement() - [](start = 0, length = 72)

I believe we shouldn't have this method in the API, it can never represent a valid syntax.